docs/UserGuide.md

```diff
@@ -158,41 +168,43 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 Example:
 * `find Software` returns `Google` and `Wise` application whose position contain software keyword.
+
 ![FindExample](images/FindExample.png)
 
 ### Editing an internship application: `edit`
 Edits an existing application in the CinternS.
 
-Format: `edit INDEX [n/COMPANY] [ct/CONTACT] [e/EMAIL] [p/POSITION] [d/DATE_APPLIED] [t/TAG]...`
+Format: `edit INDEX [n/COMPANY] [ct/CONTACT] [e/EMAIL] [p/POSITION] [d/DATE_APPLIED] [s/STATUS] [t/TAG]...`
 
 * Edits the application at the specified `INDEX`. 
 * The index refers to the index number shown in the displayed application list. 
 * The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* Changes to the company name and the position will also be reflected in the interview list.
 * When editing tags, the existing tags of the application will be removed i.e adding of tags is not cumulative.
 * You can remove all the application’s tags by typing `t/` without specifying any tags after it.
 
 Examples:
 
 * `edit 1 ct/91234567 e/grab@example.com` edits the contact and email of the 1st application to be 91234567 and grab@example.com respectively.
-* `edit 2 c/Garena t/` edits the company of the 2nd application to be `Garena` and clears all existing tags.
+* `edit 2 c/Garena t/` edits the company of the 2nd application and the corresponding interview to be `Garena` and clears all existing tags.
 
 ### Adding / Editing an interview: `interview`
 Adds an interview to an existing application or edits an existing interview in an application.
 
 Format: `interview INDEX ir/ROUND id/INTERVIEW_DATE it/INTERVIEW_TIME il/LOCATION`
 
 <div markdown="span" class="alert alert-primary">:bulb: <b>Tip:</b><br>
-1. InterviewDate must be specified in the format <em>yyyy-MM-dd</em>.<br>
-2. InterviewTime must be specified in the format <em>HHmm</em>.<br>
+1. INTERVIEW_DATE must be specified in the format <em>yyyy-MM-dd</em>.<br>
+2. INTERVIEW_TIME must be specified in the format <em>HHmm</em>.<br>
```

Comment:
```suggestion
1. `INTERVIEW_DATE` must be specified in the format <em>yyyy-MM-dd</em>.<br>
2. `INTERVIEW_TIME` must be specified in the format <em>HHmm</em>.<br>
```

Reply:
Oh I did that at first but this section is with HTML tags, the `` wouldn't work.

Reply:
Oh I see, never knew that. Never mind then

Reply:
I found the fix online actually. Turns out we need to use `<code></code>` instead.